tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.TeamFoundation.Build.WebApi;
+
+namespace PipelineGenerator.Conventions
+{
+    public class WeeklyIntegrationTestingPipelineConvention : IntegrationTestingPipelineConvention
+    {
+        public WeeklyIntegrationTestingPipelineConvention(ILogger logger, PipelineGenerationContext context) : base(logger, context)
+        {
+        }
+
+        protected override string GetDefinitionName(SdkComponent component)
+        {
+            return $"{Context.Prefix} - {component.Name} - tests-weekly";
+        }
+
+        protected override Schedule CreateScheduleFromDefinition(BuildDefinition definition)
```

Comment:
This change seems to be in two PRs?

Reply:
Agreed lets separate these changes into their own PR.

Reply:
Looks like the pipeline generation changes are already part of https://github.com/Azure/azure-sdk-tools/pull/1026